18_regex/slides.md

```diff
@@ -477,24 +479,25 @@ Most regex is transferable
 
 ---
 
-```
- ____            _
-| __ )  ___  ___| |_
-|  _ \ / _ \/ __| __|
-| |_) |  __/\__ \ |_
-|____/ \___||___/\__|
-
- ____                 _   _
-|  _ \ _ __ __ _  ___| |_(_) ___ ___  ___
-| |_) | '__/ _` |/ __| __| |/ __/ _ \/ __|
-|  __/| | | (_| | (__| |_| | (_|  __/\__ \
-|_|   |_|  \__,_|\___|\__|_|\___\___||___/
 
+```
```

Comment:
:ascii-parrot:

Does this overflow in lookatme if it's zoomed in?

Reply:
ah yes :') 

maybe I should shorten it to "Dirty Things"?

Reply:
realigned the ascii art